src/model/Activity.java

```diff
@@ -0,0 +1,33 @@
+package model;
+
+import utils.Validation;
+
+public abstract class Activity {
+
+    private String title;
+    private String slug;
+    private boolean privated;
```

Comment:
Acho que `private` mesmo, não?

```suggestion
    private boolean private;
```

Reply:
É que `private` é palavra reservada, daí tinha visto na internet `privated` como um sinônimo para private. De toda forma, como você comentou abaixo sobre o mesmo conceito em três nomes distintos, vou mudar isso e unificar :)

Reply:
Putz! Ah, é! Palavra reservada hehehe Entendi...